infra.bs

```diff
@@ -1694,6 +1694,16 @@ unordered one, since interoperability requires that any developer-exposed enumer
 contents be consistent between browsers. In those cases where order is not important, we still use
 ordered sets; implementations can optimize based on the fact that the order is not observable.
 
+<div algorithm>
+<p>To <dfn export for=set>create a set</dfn>, given a [=/list=] |input|:
+
+<ol>
+ <li><p>Let |result| be an empty [=/set=].
+ <li><p>For each [=list/item=] |item| in |input|, [=set/append=] |item| to |result|.
```

Comment:
for each x of y*

Reply:
Thanks, changed to `for each x of y`, also linked `for each`.